resources/guidelines/testing/store/quality-guidelines-plugins/index.md

```diff
@@ -1,356 +1,373 @@
 ---
 nav:
-  title: Quality guidelines for apps in plugin system
+  title: Quality guidelines for apps in the plugin system
   position: 10
 
 ---
 
 # Quality Guidelines for the Plugin System in the Shopware Store
 
 > **Changelog**
->> 01/08/24: [Added - Message queue](../quality-guidelines-plugins/#message-queue)
 >
->> 26/07/23: [Added - Identical name rule](../quality-guidelines-plugins/#every-app-based-on-the-plugin-system)
->
->> 06/09/23: [Added - Rules for own composer dependencies](../quality-guidelines-plugins/#own-composer-dependencies)
->
->> 26/07/23: [Added - Name preset according to new naming scheme](../quality-guidelines-plugins/#every-app-based-on-the-plugin-system)
->
->> 18/07/23: [Compiled code](../quality-guidelines-plugins/#checklist-for-app-testing)
->
->> 12/04/23: [Check for a functional comparison with functions from the Rise or above edition.](../quality-guidelines-plugins/#every-app-based-on-the-plugin-system)
->
->> 14/02/23: [Added new STP tracking - External technology apps/STP apps.](../quality-guidelines-plugins/#external-technology-shopware-technology-partner-stp-apps)
->
->> 11/07/22: [Added a list of all Blockers - Changes to the autom. codereview 1.st Oct. 2022](../quality-guidelines-plugins/#sonarcube-rules-status-blocker)
->
->> 26/10/21: [Added safe your app idea and get a preview in the store.](../quality-guidelines-plugins/#every-app-based-on-the-plugin-system)
```

Comment:
Why are all these being removed ?

Reply:
You can leave it like it is, but in this case, you have to change the deep links.